package.json

```diff
@@ -6,7 +6,11 @@
   "license": "MIT",
   "homepage": "https://github.com/giakki/uncss",
   "scripts": {
-    "test": "grunt test"
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "eslint": "eslint \"**/*.js\"",
+    "mocha": "nyc --reporter=html --reporter=text mocha tests/*.js --reporter spec --slow 7500 --timeout 25000",
```

Comment:
And how about the `coverage` command? Should it use `--reporter=text-lcov` or `--reporter=html-cov`?